app/views/organizations/edit.html.erb

```diff
@@ -114,10 +114,16 @@
                   <% end %>
 
                   <%= f.input_field :logo, as: :file %>
-                  <p class="help-block">
-                    Logo should be a 4:1 ratio of width / height (default image is 763 x 188 pixels).
-                    Only jpeg/png format is supported.
-                  </p>
+                  <div>
+                    <p class="help-block">
+                      Logo should be a 4:1 ratio of width / height (default image is 763 x 188 pixels).
+                      Only jpeg/png format is supported.
+                    </p>
+                    <p class="help-block warning-text">
+                      <b>Warning:</b> Images should be no more than 1M in size. Uploading a larger image <br> 
```

Comment:
This should say "1 MB" - "1M" means "one million" without a unit.  
Also, I'm not sure this warning text is necessary since we are flat out rejecting any file that's bigger than this. If we do want text, it probably makes sense to add it to the help block above, e.g. "Only jpeg/png format is supported. Images must be under 1 MB in size."

Reply:
I think we do want the text -- though I do take your point that it needn't be a warning if we are stopping them (I think stopping them was a bonus round )

Reply:
I think we can just use the wording I had above - "because of performance reasons" doesn't really say much to the end user.

I think if we have the backend validation and the info text, we shouldn't need additional client validation.

Reply:
@dorner I have made the requested changes, kindly check.